cloudfoundry/provider/resource_app.go

```diff
@@ -568,3 +577,64 @@ func (r *appResource) Delete(ctx context.Context, req resource.DeleteRequest, re
 func (r *appResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
 	resource.ImportStatePassthroughID(ctx, path.Root("id"), req, resp)
 }
+
+func getAppLogTrace(ctx context.Context, r *appResource, desiredState AppType, curTime time.Time) []string {
+
+	EnableCFAppLogTrace := os.Getenv("ENABLE_CF_APP_LOG_TRACE")
```

Comment:
Please add this to the documentation as well... maybe in the schema markdown, this point can be added